README.md

```diff
@@ -22,5 +22,7 @@ JIRA_TOKEN="..." junit2jira \
   -build-link "https://..." \
   -build-tag "$STACKROX_BUILD_TAG|$GITHUB_SHA" \
   -job-name "$JOB_NAME|$GITHUB_WORKFLOW" \
-  -orchestrator "$ORCHESTRATOR_FLAVOR"
+  -orchestrator "$ORCHESTRATOR_FLAVOR" \
+  -timestamp $(date --rfc-3339=seconds)
```

Comment:
As the list is getting longer, it would be good to show somehow which arguments are required and which are optional..

Reply:
Added help output in readme